docs/interpreter/spark.md

```diff
@@ -365,6 +365,51 @@ myScalaDataFrame = DataFrame(z.get("myScalaDataFrame"), sqlContext)
   </div>
 </div>
 
+### Object Interpolation
+Some interpreters can interpolate object values from `z` into the command string by using the 
```

Comment:
question: perhaps this is talked about but the meaning of "command string" isn't very clear to me as an user.
Are we referring to the paragraph text?

Reply:
"command" and "command string" used everywhere in my original documentation has been changed to "paragraph text".